MonoGame.Framework/Graphics/Texture2D.DirectX.cs

```diff
@@ -403,7 +403,20 @@ internal override SharpDX.Direct3D11.Resource CreateTexture()
             }
             if (_shared)
                 desc.OptionFlags |= SharpDX.Direct3D11.ResourceOptionFlags.Shared;
-
+            
+            if (_format == SurfaceFormat.Dxt1 ||
+                _format == SurfaceFormat.Dxt3 ||
+                _format == SurfaceFormat.Dxt5)
+            {
+                if (width % 4 != 0 || height % 4 != 0)
+                {
+                    // Note that the sharpdx call immediately below this will in fact fail
+                    // in this case. However it fails with an extremely unhelpful message. 
+                    // The message we write here matches the one given by XNA.
+                    throw new System.ArgumentException("DXT compressed texture sizes must be multiples of four.");
+                }
+            }
```

Comment:
@tgjones @KonajuGames 

Maybe this should be higher up in the common code?  I would think even if using DXT on OpenGL or Android we would still need the same validation.  Correct?